x-pack/plugins/observability_onboarding/public/application/app.tsx

```diff
@@ -55,19 +57,42 @@ export const breadcrumbsApp = {
 };
 
 function App() {
+  const customLogRoutesPaths = Object.keys(customLogsRoutes);
+
   return (
     <>
       <Routes>
-        {Object.keys(routes).map((key) => {
+        {Object.keys(baseRoutes).map((key) => {
           const path = key as keyof typeof routes;
           const { handler, exact } = routes[path];
           const Wrapper = () => {
             return handler();
           };
+
           return (
             <Route key={path} path={path} exact={exact} component={Wrapper} />
           );
         })}
+        <Route exact path={customLogRoutesPaths}>
```

Comment:
This looks good, but maybe in a future enhancement we can incorporate these routes with the existing wizard Provider that's already exported. Not even sure if it's possible, but it would be much more easy for a consumer to get those routes without having to loop thru them to render the routes.

Reply:
Agree but don't know if that's possible. I could check about this for a future iteration